apps/web/lib/d/[link]/[slug]/getServerSideProps.tsx

```diff
@@ -85,16 +92,17 @@ async function getUserPageProps(context: GetServerSidePropsContext) {
       }
     }
 
+    name = profileUsername || username;
```

Comment:
If theres a profileUsername we use this as priority to find the user. (this is how its handled inside of the userRepository.